snp_scanning/scripts/scan_motion_plan_from_file_node

```diff
@@ -20,18 +36,31 @@ class ScanMotionPlanFromFileNode(Node):
             with open(self.get_parameter('scan_trajectory_file').value, "r") as file_input:
                 scan_traj = yaml.safe_load(file_input)
 
-            response.motion_plan.header.stamp = Time(sec=int(scan_traj['header']['stamp']['sec']),nanosec=int(scan_traj['header']['stamp']['nanosec']))
-            response.motion_plan.header.frame_id = scan_traj['header']['frame_id']
-            response.motion_plan.joint_names = scan_traj['joint_names']
-            for point_val in scan_traj['points']:
-                point = JointTrajectoryPoint()
-                point.positions = point_val['positions']
-                point.velocities = point_val['velocities']
-                point.accelerations = point_val['accelerations']
-                point.effort = point_val['effort']
-                point.time_from_start = Duration(sec = int(point_val['time_from_start']['sec']),
-                                                 nanosec= int(point_val['time_from_start']['nanosec']))
-                response.motion_plan.points.append(point)
+            points = scan_traj['points']
+
+            header = Header()
+            header.stamp = Time(sec=int(scan_traj['header']['stamp']['sec']),nanosec=int(scan_traj['header']['stamp']['nanosec']))
+            header.frame_id = scan_traj['header']['frame_id']
+
+            # Approach
+            response.approach.header = header;
+            response.approach.joint_names = scan_traj['joint_names']
+            for i in range(0, 2):
+              response.approach.points.append(create_point(points[i]))
+
+            # Process
+            # TODO: adjust time
```

Comment:
What does "adjust time" mean in this context?

Reply:
It's already accounted for in `time_offset` [here](https://github.com/marip8/scan_n_plan_workshop/blob/cf60b557fa079c01d16ed607cf46b3bd997c7fc3/snp_scanning/scripts/scan_motion_plan_from_file_node#L13). I'll remove the comments